changelogs/unreleased/gh_5997_get_rid_of_schema_max_id.md

```diff
@@ -0,0 +1,4 @@
+## feature/core
+
+* Tarantool does not use `_schema.max_id` anymore, so this field is
+  dropped (gh-5997).
```

Comment:
`max_id` is [documented](https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_schema/) so we need to file a ticket to update the doc. Please add a doc bot request.

Reply:
Done